networking/route/pmtu/metadata

```diff
@@ -0,0 +1,12 @@
+[General]
+name=networking/route/pmtu
+owner=Jianlin Shi <jishi@redhat.com>
+description=Networking pmtu tests 
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+softDependencies=net-tools;tcpdump;nmap;ethtool;iproute;nc;nmap-ncat;socat;libtool;bind-utils;bridge-utils;libteam;teamd;traceroute;expect;pexpect;python3-pexpect;bc;wget;bzip2;gcc;git;patch;rsync;texinfo
```

Comment:
Does it mean dependencies instead of softDependencies?  :-)

Reply:
Since there are more than 20 packages are required, I don't know what packages will break the process of handling dependencies, so use `softDependencies` instead of `dependencies` to make sure the test can continue. You know, if a package listed in `dependencies` cannot be installed, it will abort in handling `metadata` phase. But if a package listed in `softDependencies` (e.g. bridge-utils) cannot be installed, it will not.

Reply:
Yes,it is true. It is taking extra time (trials and errors) to know which packages  should be in  softDependencies.
I am thinking  these packages  should be in softDependencies: net-tools;bridge-utils;nmap-ncat;libteam;teamd;pexpect;python3-pexpect. 

@rasibley ,
should we need to put extra works to find packages that should be in sofDependencies metadata when involving many packages?
Either way, should be good.

Reply:
IMO yes we should take a look at the previous harness.log where we use Makefiles for supported releases to determine which packages didn't install. I'm hoping this isn't too painful, but we should try to do the right thing, otherwise this could lead to confusion if a non-rh contributor is running our tests.

Reply:
To figure out what packages should be listed in `softDependencies`, I reserved two hosts (the 1st one's os is RHEL7, the 2nd one's os is RHEL8), then I had a try to install packages (e.g. `bc;bind-utils;bzip2;ethtool;expect;bridge-utils;pexpect;python3-pexpect`), if any one among them could not be installed on either RHEL7 or RHEL8, it should be listed in `oftDependencies`, e.g. `bridge-utils;pexpect;python3-pexpect`. Let's wait for the test results triggered off by cki-bot...